### DISCUSSION TITLE
Private Organizations Visible only to its members?

### DISCUSSION THREAD

**Discussion by arnav-kr** [community/community#136300] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

So I had joined an org in the past https://github.com/aovim, today I randomly opened the URL in incognito, found it returns a 404 (it does show up in my profile though)
![image](https://github.com/user-attachments/assets/561e3e3e-4297-4d2d-a0cf-38db51ac269d)

But when logged in with my account, it does show up
<img width="959" alt="image" src="https://github.com/user-attachments/assets/8db8457f-aa8d-428a-8af2-9e0d483304dc">

I asked a few other people to check with their account logged in, and it was 404 for them too.

Checked the GitHub API [https://aphttps://api.github.com/orgs/aovimi.github.com/orgs/aovim
](https://api.github.com/orgs/aovim) it's normal, the data show up

afaik GitHub doesn't have a feature for orgs that are only visible to its members and not the other outside people.

or if I am wrong, can you tell, how can I achieve the same

**Comment by pierrelouisguichard** [comment]
Hey,

If an organization is set to private, non-members will see a 404 error page when trying to access it. To allow anyone to view the organization at https://github.com/aovim, you need to make the organization public. You can do this by going to Settings > Profile > Public and then clicking Save.

I hope this addresses your question and resolves the issue.

**Reply by arnav-kr** [reply]
Hi, thanks for a response
but I think you misunderstood it, the visibility feature for organizations doesn't exist 
![image](https://github.com/user-attachments/assets/cd463a63-4444-4170-9038-96a3d2cd0400)

for reference, here is a discussion on the visibility feature for github orgs
https://github.com/orgs/community/discussions/23157

**Comment by arnav-kr** [comment, marked as answer]
most probably shadow banned
